src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -68,6 +69,11 @@ internal sealed class SourcePropertySymbol : PropertySymbol, IAttributeTargetSym
             _syntaxRef = syntax.GetReference();
             _refKind = syntax.Type.GetRefKind();
 
+            if (_refKind == RefKind.RefReadOnly)
+            {
+                bodyBinder.Compilation.EnsureIsReadOnlyAttributeExists(diagnostics, syntax.Type.Location, modifyCompilation: true);
```

Comment:
@AlekseyTS For this specific symbol, we do bind in the constructor (check line 241):
`_lazyType = this.ComputeType(bodyBinder, syntax, diagnostics);`
Can you please explain the difference?

---
In reply to: [115105712](https://github.com/dotnet/roslyn/pull/18715#discussion_r115105712) [](ancestors = 115105712)